src/commands.ts

```diff
@@ -381,6 +381,23 @@ export function registerCommands(
 		),
 	);
 
+	context.subscriptions.push(
+		vscode.commands.registerCommand(
+			'pr.pushAndCreate',
+			async (args?: any | Repository) => {
+				if (isSourceControl(args)) {
+					const reviewManager = await chooseReviewManager(args.rootUri.fsPath);
+					if (reviewManager) {
+						if (args.state.HEAD?.upstream) {
+							await args.push();
+						}
+						reviewManager.createPullRequest();
```

Comment:
Does this part mean that even if the push fails, the PR might be created?

Reply:
If the push fails there will be an uncaught exception, so the PR will not be created.